.github/autosubmit.yml

```diff
@@ -0,0 +1,16 @@
+default_branch: 'main'
+issues_repository:
+  owner: 'flutter'
+  repo: 'flutter'
+auto_approval_accounts:
+  - 'skia-flutter-autoroll'
```

Comment:
Are all of these bots needed for Cocoon?

Reply:
That I am not sure about. I will check.

Reply:
We need only `dependabot` for cocoon.